File: C/inout.c
#include <stdio.h>

int main() {
    char name[50];
    
    printf("Please enter your name: ");
    
    // Read input from user
    fgets(name, 50, stdin);
    
    // Print the input back to the user
    printf("Hello, %s", name);
    
    return 0;
}

File: Java/inout.java
import java.util.Scanner;

public class InputOutput {
    public static void main(String[] args) {
        // Create a Scanner object to read input
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Please enter your name: ");
        
        // Read the user input
        String name = scanner.nextLine();
        
        // Display the input
        System.out.println("Hello, " + name);
        
        // Close the scanner
        scanner.close();
    }
}